src/gtfs/read.rs

```diff
@@ -1741,25 +1741,31 @@ mod tests {
             );
             assert_eq!(
                 vec![
-                    ("my_prefix:route_1", "my_prefix:agency_1", "Bus"),
-                    ("my_prefix:route_2", "my_prefix:agency_1", "Bus"),
+                    ("my_prefix:route_1", "my_prefix:agency_1", "Bus", None, None),
+                    ("my_prefix:route_2", "my_prefix:agency_1", "Bus", None, None),
                 ],
                 extract(
                     |obj| (
                         obj.id.as_str(),
                         obj.network_id.as_str(),
                         obj.commercial_mode_id.as_str(),
+                        obj.forward_direction.as_ref().map(|e| e.as_str()),
```

Comment:
line.foward_direction, line.backward_direction and route.destination_id (= None) are not managed in gtfs2ntfs
I added tests here to be sure that it will fail once implemented